### DISCUSSION TITLE
error with agent

### DISCUSSION THREAD

**Discussion by BarryBahrami** [whiteducksoftware/azure-devops-build-agents-community#10] [Q&A] [answered]
I installed your windows vm agent for azure devops and am getting this error.  can you advise please?  thank you
![agentissue](https://github.com/whiteducksoftware/azure-devops-build-agents-community/assets/65618672/aeda3262-dc87-4e1a-bd4e-ce3de3441244)

**Comment by ndeeH** [comment, marked as answer]
Hi @BarryBahrami,
the latest node.js version is already installed.
You have to install the needed npm package manager version in the created vm.

Alternatively you could use our scaleset version where you can use your own preconfigured images containing the needed package manager.

Come back here, if you have further questions.

Bye, Andi
